README.md

```diff
@@ -38,90 +38,132 @@
 - Classification and attribute inference of nodes or edges;
 - Link prediction;
-- Interpretation of node classification through calculated importances of edges and neighbour nodes for selected target nodes [8].
-
-We provide [examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) of using `StellarGraph` to solve such tasks using several real-world datasets.
+- Interpretation of node classification through [calculated importances of edges and neighbour nodes for selected target nodes](https://medium.com/stellargraph/https-medium-com-stellargraph-saliency-maps-for-graph-machine-learning-5cca536974da) [8].
 
+Graph-structured data represent entities as nodes (aka vertices) and relationships between them as edges (aka links), along with associated data as attributes. For example, a graph can contain people as nodes and friendships between them as links, with data like a person's age and the date a friendship was established. StellarGraph supports analysis of many kinds of graphs:
 
-## Guiding Principles
+- homogeneous (with nodes and links of one type),
+- heterogeneous (with more than one type of nodes and/or links)
+- knowledge graphs (extreme heterogeneous graphs with thousands of types of edges)
+- graphs with or without data associated with nodes
+- graphs with edge weights
 
-StellarGraph uses the [Keras](https://keras.io/) API as implemented in the [TensorFlow](https://tensorflow.org/) library and adheres to the same
-guiding principles as Keras: user-friendliness, modularity, and easy extendability. Modules and layers
-of StellarGraph library are designed so that they can be used together with
-standard Keras layers and modules, if required. This enables flexibility in using existing
-or creating new models and workflows for machine learning on graphs.
+StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras high-level API](https://www.tensorflow.org/guide/keras), as well as [Pandas](https://pandas.pydata.org) and [NumPy](https://www.numpy.org). It is thus user-friendly, modular and extensible. It interoperates smoothly with code that builds on these, such as the standard Keras layers and [scikit-learn](http://scikit-learn.github.io/stable), so it is easy to augment the core graph machine learning algorithms provided by StellarGraph.
 
 ## Getting Started
 
-To get started with StellarGraph you'll need data structured as a homogeneous or heterogeneous graph, including
-attributes for the entities represented as graph nodes.
-[NetworkX](https://networkx.github.io/) is used to represent the graph and [Pandas](https://pandas.pydata.org/)
-or [Numpy](https://www.numpy.org/) are used to store node attributes.
+[The numerous detailed and narrated examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) are a good way to get started with StellarGraph. There is likely to be one that is similar to your data or your problem (if not, [let us know](#getting-help)).
```

Comment:
At some point in future, we should expand this to link to some 'tutorial' content - the readme page for the demos doesn't tell people where to start; it's just an index.  We could even link to our machine learning on graphs course?

Reply:
> we should expand this to link to some 'tutorial' content

covered by #992 actually

Reply:
I think we should either make the `demos` landing page have guidance, to avoid making this document too huge, or switch to doing it all on read-the-docs, which case this link would switch to point there.

Reply:
@timpitman thoughts?

Reply:
I like the idea of improving the demos landing page (ie README in the demos folder).  That could even optionally be embedded in our readthedocs. documentation.